server/main.go

```diff
@@ -1,28 +1,42 @@
 package main
 
 import (
+	"database/sql"
 	"flag"
-	"github.com/sirupsen/logrus"
+	_ "github.com/mattn/go-sqlite3"
+	log "github.com/sirupsen/logrus"
 	"net/http"
 	"strings"
+	"time"
+	"unshort.link/blacklist"
 )
 
-//go:generate curl -o blacklist.txt https://hosts.ubuntu101.co.za/domains.list
 //go:generate go get -u github.com/programmfabrik/esc
-//go:generate esc -private -local-prefix-cwd -pkg=main -o=static.go static/ blacklist.txt
```

Comment:
I assume you did not clean build? As if you remove this complete line the `go build` does not work anymore, as the static files are not build into static.go anymore. 
This line should be keept but, blacklist.txt removed::
`//go:generate esc -private -local-prefix-cwd -pkg=main -o=static.go static/` than the generate and build works as expected

Reply:
For reproducing that on your machine: 

1) Remove `static.go` 
2) Run `go generate ./...`
3) Run `go buid`

Reply:
Good point I will change it